glide.lock

```diff
@@ -318,6 +318,11 @@ imports:
   subpackages:
   - pkg/apis/apiextensions
   - pkg/apis/apiextensions/v1beta1
+  - pkg/client/clientset/clientset
```

Comment:
the glide.yaml didn't add new packages, do you know why these new dependencies got added to the glide.lock?